pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -133,6 +133,9 @@ const (
 	// using an SSM GetParameter query
 	NodeImageResolverAutoSSM = "auto-ssm"
 
+	// EksctlVersionTag defines the version of eksctl which is used to provision or update EKS cluster
+	EksctlVersionTag = "alpha.eksctl.io/eksctl-version"
```

Comment:
The API version is `v1alpha5` but eksctl's version is `0.13.0`... If we are using eksctl's version, should we still use tags that say `alpha`? :confused: It's probably good to keep all of them with the same convention but I'm wondering if we should graduate them to proper versions (not necessarily in this PR)... What do you guys think?